release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1263,6 +1263,24 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-15-dev-console-bug-fixes"]
 ==== Developer Console
 
+* Previously, `BuildRun` logs were not visible in the *Logs* Tab of the BuildRun due to the recent update in the API version of the specified resources. With this update, the Logs of the `TaskRuns` were added back into the *Logs* tab of the BuildRun for both v1alpha1 as well as v1beta1 versions of the builds Operator. (link: https://issues.redhat.com/browse/OCPBUGS-29283[*OCPBUGS-29283*])
+
+* Previously, the console UI failed when a *Task* in the Pipeline Builder that was previously installed from the *ArtifactHub* was selected and an error page displayed. With this update, the console UI no longer expects optional data and the console UI no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-24001[*OCPBUGS-24001*])
+
+* Previously, the *Edit Build* and *BuildRun* options in the *Actions* menu of the Shipwright Plugin did not allow you to edit in the YAML tab. With this update, you can edit in the YAML tab. (link:https://issues.redhat.com/browse/OCPBUGS-23164[*OCPBUGS-23164*])
+
+* Previously, the console searched only for the file name `Dockerfile` in a repository to identify the repository suitable for the `Container` strategy in the Import Flows. Since other containerization tools are available, support for the the `Containerfile` file name is now suitable for the `Container` strategy. (link:https://issues.redhat.com/browse/OCPBUGS-22976[*OCPBUGS-22976*])
```

Comment:
```suggestion
* Previously, the console searched only for the file name `Dockerfile` in a repository to identify the repository suitable for the `Container` strategy in the Import Flows. Since other containerization tools are available, support for the `Containerfile` file name is now suitable for the `Container` strategy. (link:https://issues.redhat.com/browse/OCPBUGS-22976[*OCPBUGS-22976*])
```